spot-client/src/spot-tv/ui/components/meeting-status/meeting-status.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { getInMeetingStatus } from 'common/app-state';
+import { MicOff, VideocamOff } from 'common/icons';
+
+import StatusIcon from './status-icon';
+
+/**
+ * Displays Spot-TV in-meeting status, such as mute states.
+ *
+ * @extends React.Component
+ */
+export class MeetingStatus extends React.Component {
+    static propTypes = {
+        audioMuted: PropTypes.bool,
+        videoMuted: PropTypes.bool
+    };
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        return (
+            <div className = 'meeting-status'>
+                {
+                    this.props.audioMuted && (
+                        <StatusIcon
+                            icon = { <MicOff /> }
```

Comment:
Don't you wanna pass the component class instead of an instantiated component? Not sure what this causes, it just feels weird to me.

Reply:
Maybe a child instead?